### PR TITLE
Improved connect logging

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Added logging to hopefully help debug connectivity issues. Also makes the version check a bit more strict and requires 10.6.0 >=

image

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
We should add a version check on app launch too in the future, as right now you can add a server with version 10.6.x but when you update the app and it requires 10.7.x it still connects happily to a server already configured.

File: app/src/main/java/org/jellyfin/mobile/fragment/ConnectFragment.kt [hunk 1]
```diff
@@ -190,10 +196,20 @@ class ConnectFragment : Fragment() {
...
-            version[0] == PRODUCT_NAME_SUPPORTED_SINCE.first && version[1] < PRODUCT_NAME_SUPPORTED_SINCE.second -> true // Valid old version
+            // Major version is invalid
+            version[0] != PRODUCT_NAME_SUPPORTED_SINCE.first -> false
+            // Minor version is too old
+            version[1] < PRODUCT_NAME_SUPPORTED_SINCE.second -> false
             else -> true // FIXME: check ProductName once API client supports it
         }
 
-        return if (isValidInstance) httpUrl else null
+        Timber.i("Server at $urls with version ${serverInfo.version} valid: $isValidInstance")
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/fragment/ConnectFragment.kt:206]:
```suggestion
        Timber.i("Server at $httpUrl with version ${serverInfo.version} valid: $isValidInstance")
```

oops

**Comment by Maxr1998** [comment]
I don't know whether we want to leak the server url in the production version, I'd prefer a `Timber.d` for the logging calls. Rest looks good, gonna review once I'm at the PC.

**Comment by nielsvanvelzen** [comment]
We could mask the hostname but in the end the user is responsible for making sure to not leak the address. The apiclient, webview and a bunch of other components also print it.

**Comment by Maxr1998** [comment]
Also in the non-debug version?

**Comment by nielsvanvelzen** [comment]
Yup

**Review by Maxr1998 — APPROVED** [review]
LGTM, if you fix the thing you commented on, I'll merge.

**Comment by nielsvanvelzen** [comment]
I was too lazy to open Android Studio so can just squash merge it

**Comment by Maxr1998** [comment]
Hehe, ok, gonna merge once the pipeline finishes :+1:

**Comment by Maxr1998** [comment]
Effin' Azure :angry:
It builds fine for me locally, so I'll merge this anyways.

**Pull request merged**